src/utils/interfaces.ts

```diff
@@ -0,0 +1,5 @@
+export interface StatsData {
```

Comment:
I think its better to have a separate directory of all Type Aliases & Interfaces instead of keeping them inside a "constants" folder.